README.md

```diff
@@ -10,3 +10,16 @@
 ## Licence
 
 Apache 2
+
+## Releasing
+
+We don't use `zest.releaser` right now because of some problems with
+releasing a package that is not at the root of the repo (`jobs/`), but
+this could change if we figure out how.
```

Comment:
I think it's also because we don't really need pypi. It runs in a lambda that we publish as a zip on the release page

Reply:
Ah right, I forgot to mention about the release page, thanks.